independent_reserve_client-ruby.gemspec

```diff
@@ -1,36 +1,34 @@
 # coding: utf-8
 lib = File.expand_path("../lib", __FILE__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
-require "independent_reserve_client/ruby/version"
+require "independent_reserve_client/version"
 
 Gem::Specification.new do |spec|
   spec.name          = "independent_reserve_client-ruby"
```

Comment:
Since this gem is pushed to `rubygems` I think we could forgo with the `-ruby` suffix. Probably rename the `spec.name` and the `independent_reserve_client-ruby.gemspec` file itself too? Just like in `quoine_client`

Reply:
Got it. Will change this.